Autopilot/Path Manager/MainBatterySensor.c

```diff
@@ -6,40 +6,34 @@
  */
 #include "main.h"
 #include "../Common/Common.h"
-#include "voltageSensor.h"
+#include "MainBatterySensor.h"
```

Comment:
I originally named them MainBattery and ExtBattery, but removed external battery because its more complicated. MainBattery is the main power to the PicPilot, and External Battery is the motor batteries (external because it has nothing to do with the picpilot). I can add some comments to clarify this, and it will be more clear when I add the external battery sensor.